decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -245,8 +245,8 @@ def current_user_can_visit_meeting?(user)
       def iframe_access_level_allowed_for_user?(user)
         case iframe_access_level
         when "all"
-          true
-        when "signed_in"
+          false
+        when "signed_in" && "registered"
```

Comment:
No need to change this, as it's working the access control for the iframe 

```suggestion
          true
        when "signed_in"
```

Reply:
(This is making the failing spec in the pipeline)

Reply:
Thanks.